sdk/communication/Azure.Communication.CallAutomation/src/autorest.md

```diff
@@ -10,7 +10,7 @@ model-namespace: false
 tag: package-2023-10-03-preview
 
 require:
-    - https://github.com/Azure/azure-rest-api-specs/blob/e512ec327a8a5c8a5edc3b72a5aafa4425d16248/specification/communication/data-plane/CallAutomation/readme.md
+    - https://github.com/amariwest-msft/azure-rest-api-specs/blob/e8971b3bcc351ac3de38e2f5446bdc036d02471a/specification/communication/data-plane/CallAutomation/readme.md
```

Comment:
This is pointing to personal repo - is it possible to update latest specs in official repo branch? (i.e. [alpha3](https://github.com/Azure/azure-rest-api-specs/tree/callautomation/alpha3))

Reply:
Updated specs